Makefile

```
@@ -399,9 +399,27 @@ install-tools:
 	$(GO) install github.com/vektra/mockery/v2@v2.14.0
 	$(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.52.1
 	$(GO) install mvdan.cc/gofumpt@latest
+	$(GO) install github.com/josephspurrier/goversioninfo/cmd/goversioninfo@v1.4.0
+
+.PHONY: install-test-tools
+install-test-tools:
+    $(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.52.1
+    $(GO) install mvdan.cc/gofumpt@latest
+
+.PHONY: install-build-tools
+install-build-tools:
+    $(GO) install github.com/josephspurrier/goversioninfo/cmd/goversioninfo@v1.4.0
+    
+
+PHONY: install-build-ci
+install-build-ci: install-build-tools
+
+.PHONY: install-test-ci
+install-test-ci: install-test-tools 
 
 .PHONY: install-ci
-install-ci: install-tools
+install-ci: install-tools install-test-ci install-build-ci
```

```suggestion
install-ci: install-build-tools
```
and no need to change all workflows, only change the two test workflows to call into `build-test-tools` target

---

And for the test workflow